pinot-broker/src/test/java/org/apache/pinot/broker/requesthandler/LiteralOnlyBrokerRequestTest.java

```diff
@@ -211,6 +219,59 @@ public void testBrokerRequestHandlerWithAsFunction()
     Assert.assertEquals(brokerResponse.getResultTable().getRows().get(0)[1].toString(),
         "key1=value 1&key2=value@!$2&key3=value%3");
     Assert.assertEquals(brokerResponse.getTotalDocs(), 0);
+
+    request = JsonUtils.stringToJsonNode(
+        "{\"sql\":\"SELECT toBase64('hello!') AS encoded, " + "fromBase64('aGVsbG8h') AS decoded\"}");
+    requestStats = Tracing.getTracer().createRequestScope();
+    brokerResponse = requestHandler.handleRequest(request, null, requestStats);
+    System.out.println(brokerResponse.getResultTable());
```

Comment:
(nit) this code can be made slightly more readable imho by doing get on the resultTable , dataSchema and getRows exactly once.

Reply:
Done!